### ISSUE TITLE
no more underground oceans after updating :(

### ISSUE THREAD

**Issue by MCJediYoda** [Reoseah/UndergroundOcean#3]
so I updated to the latest 1.1.0 alpha and I no longer have underground ocean.  This is what I really want for my hardcore world.  After updating, I have more like caves with a little bit of water.  I went back to 1.0.1-b1 and recreated the same world and I have oceans again.  Is this intentional?  If so, I will try playing with the configuration file to see if I can somehow get my world looking like its underwater again.

**Comment by Reoseah** [comment]
Yes, I changes default values a bit. I think this is more suitable for normal gameplay while previous version slightly disrupts it. They are fine if they can be major part of game play, but not as decoration. 
However, I don't like that this lakes turned to have same deepness everywhere... I'm going to add additional 2-dimensional noise map and use it as mask for deepness. Configs will have two values for max height/deepness. 

In 1.1, I think, changing configuration can return oceans. However, I have quite "optimized" code for underground lakes. (And I slightly afraid of moving all constants to config, they may just overwhelm user, and it is hard to describe what they are doing. )

**Comment by MCJediYoda** [comment]
Fair enough.  I am going to use the previous version for now.  Those disruptions actually made my hardcore mudpack worlds really challenging and interesting.  I hope you can return the oceans at least via config :)  Thanks alot

**Comment by Reoseah** [comment]
Essentially the main change of update is using additional noise map to "mask" oceans so that they are not filling entire level of world, as were requested on Curse page:
https://minecraft.curseforge.com/projects/undergroundocean?comment=9

I haven't added any config to this mask... So, oceans will always be rare, even with configuration turned to make them as large as possible, they will be still limited by additional noise... And I changed constant that sets how fast how fast caves are halting after exceeding height limit. Current lager constants produce low flat ceiling instead of nether-like caverns.

Also. Old version has broken config. I had copy-pasted code and simply forgot to change names of variables.

**Comment by Reoseah** [comment]
I will try to return capabilities of creating old oceans.
